vars/githubscm.groovy

```diff
@@ -275,3 +275,28 @@ def findAndStageNotIgnoredFiles(String findNamePattern){
     git status
     """
 }
+
+def setDefaultBranch(String repo, String defaultBranch, String author, String credentialId = 'kie-ci', String ghPath = "../gh") {
```

Comment:
sorry, can you please explain what this means? Thanks

Reply:
This is a path to github api utility gh that needs to be installed in jenkins node before this call.
You can see the installation here: https://github.com/kiegroup/optaplanner/blob/bf3b9adb887a58b63cadd0fed96b67894d6effaf/Jenkinsfile.promote#L454 

I need it because I need to switch default branch for a while to reset stage branch rather then merge it